tests/conftest.py

```diff
@@ -1,20 +1,17 @@
-# Add parent directory to module path
-import os, sys
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
This was used as a bit of a hack to get the imports to work here. But that's been resolved now, since all the source files were moved into a separate folder.